cabal.project

```diff
@@ -0,0 +1,4 @@
+packages: .
+
+package tomland
+  flags: +build-readme +build-play-tomland
```

Comment:
I have added this config, so the `cabal run readme` command documented in `README.md` continues to work as expected.